tests/smoke/Commands/compile.smoke.yaml

```diff
@@ -11,6 +11,40 @@ tests:
         JUVIX_FILE
     exit-status: 0
 
+  - name: hello-world-no-arg
+    command:
+      shell:
+        - bash
+      script: |
+        cd ./examples/milestone/HelloWorld
+        juvix compile
+        ./HelloWorld
+    exit-status: 0
+    stdout: |
+      hello world!
+
+  - name: hello-world-no-arg-error
+    command:
+      shell:
+        - bash
+      script: |
+        temp=$(mktemp -d)
+        trap 'rm -rf -- "$temp"' EXIT
+        cd ./examples/milestone/
+        cp -r HelloWorld "$temp"
+        cd "$temp/HelloWorld"
+        if command -v gsed >/dev/null 2>&1; then
```

Comment:
Now that Jonathan has done https://github.com/anoma/juvix/pull/2123 we should remove the gsed workaround.

Reply:
I've reverted this change